worker/agentconfigupdater/manifold_test.go

```diff
@@ -269,13 +373,13 @@ func (mc *mockConfig) SetStateServingInfo(info params.StateServingInfo) {
 
 func (mc *mockConfig) MongoMemoryProfile() mongo.MemoryProfile {
 	if mc.profile == "" {
-		return mongo.MemoryProfileLow
+		return controller.DefaultMongoMemoryProfile
 	}
-	return mc.profile
+	return mongo.MemoryProfile(mc.profile)
```

Comment:
if this is just a cast, it can still lead to values that aren't part of the enumeration

Reply:
Sure, but here we don't care, and explicilty use that ability.